### PR TITLE
Release of v15.0.2

### PR DESCRIPTION
**Pull request by frouioui**
Includes the release notes and tag commit for the v15.0.2 release.

### CODE REVIEW

**Comment by frouioui** [comment]
⚠️ This PR **MUST** be merged using `Merge pull request` instead of `Squash and merge`. This is to avoid losing the release commit on the `release-15.0` branch after the merge.

**Review by rsajwani — APPROVED** [review]
LGTM

File: doc/releasenotes/15_0_2_release_notes.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Release of Vitess v15.0.2
+## Major Changes
+
+### Upgrade to `go1.18.9`
+
+Vitess `v15.0.2` now runs on `go1.18.9`.
+The patch release of Go, `go1.18.9`, was one of the main reasons for this release as it includes an important security fixe to `net/http` package, which is use extensively by Vitess.
```
Comment (deepthi) [inline doc/releasenotes/15_0_2_release_notes.md:7]:
```suggestion
The patch release of Go, `go1.18.9`, was one of the main reasons for this release as it includes an important security fix to `net/http` package, which is used extensively by Vitess.
```

**Review by deepthi — APPROVED** [review]
Found a couple of typos, otherwise LGTM

File: doc/releasenotes/15_0_2_changelog.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Changelog of Vitess v15.0.2
+
+### Bug fixes 
+#### Query Serving
+ * Online DDL: fix 'vtctlclient OnlineDDL' template queries [#11889](https://github.com/vitessio/vitess/pull/11889)
+ * Fix CheckMySQL by setting the correct wanted state [#11895](https://github.com/vitessio/vitess/pull/11895)
+ * bugfix: allow predicates without dependencies with derived tables to be handled correctly [#11911](https://github.com/vitessio/vitess/pull/11911)
+ * [release-15.0] Fix sending a ServerLost error when reading a packet fails (#11920) [#11930](https://github.com/vitessio/vitess/pull/11930)
+ * Skip `TestSubqueriesExists` during upgrade-downgrade tests [#11953](https://github.com/vitessio/vitess/pull/11953)
```
Comment (systay) [inline doc/releasenotes/15_0_2_changelog.md:9]: Should we really list CI config changes in the release notes?

Comment (frouioui) [reply doc/releasenotes/15_0_2_changelog.md:9]: Optimally no, but as of now we don't have a system to filter PRs in the changelog.

Comment (frouioui) [reply doc/releasenotes/15_0_2_changelog.md:9]: However, we should manually go through the summary during the release and clean up irrelevant rows

**Pull request merged**